esphome/components/http_request/__init__.py

```diff
@@ -98,6 +102,8 @@ async def to_code(config):
     var = cg.new_Pvariable(config[CONF_ID])
     cg.add(var.set_timeout(config[CONF_TIMEOUT]))
     cg.add(var.set_useragent(config[CONF_USERAGENT]))
+    if not config.get(CONF_ESP8266_DISABLE_SSL_SUPPORT, False):
```

Comment:
```suggestion
    if not config[CONF_ESP8266_DISABLE_SSL_SUPPORT]:
```
No need to repeat the default here.

Reply:
Fixed.

Reply:
Unfortunately it does not work for `esp32`, since in this context it is not considered to be defined by defualt, thus why we do `.get(..., False)`. I will revert it back. I see tests for `tests/test1.yaml` to break.

Reply:
Oh, you're right about that. Sorry.

Reply:
I'm slightly tired of changing this over and over :)